e-linds/Update installation.md with namespace context (#36011)
DataDog/documentation: content/en/containers/kubernetes/installation.md

   - [Datadog Operator][9] (recommended): a Kubernetes [operator][10] that you can use to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options.
   - [Helm][11]
   - Manual installation. See [Manually install and configure the Datadog Agent with a DaemonSet][12]
  

> <div class="alert alert-info">Note: If you plan to implement APM via <a href="https://docs.datadoghq.com/containers/kubernetes/apm">Single Step Instrumentation</a> in your Kubernetes environment, make sure to install the Datadog Agent in its own namespace. SSI will _not_ instrument pods in the same namespace as the Datadog Agent.  </div>

Reviewer: A few issues on this line:
- `_not_` won't render as italic inside a raw HTML block — reads fine without emphasis
- "will not" → "does not" (present tense)
- "via" → "with" (word list)
- Internal links should use relative URLs, not the full `docs.datadoghq.com` domain
- "Note:" is redundant inside an `alert-info` callout
- Spell out the abbreviation on first mention: `Single Step Instrumentation (SSI)`
- "make sure to install" → direct imperative
- Trailing double space before closing tag

```suggestion
<div class="alert alert-info">If you plan to implement APM with <a href="/containers/kubernetes/apm">Single Step Instrumentation (SSI)</a> in your Kubernetes environment, install the Datadog Agent in its own namespace. SSI does not instrument pods in the same namespace as the Datadog Agent.</div>
```
